src/toolbars/SelectionBar.cpp

```diff
@@ -59,8 +60,11 @@ with changes in the SelectionBar.
 #include "../ProjectSettings.h"
 #include "ViewInfo.h"
 #include "AllThemeResources.h"
+#include "../widgets/AButton.h"
 #include "../widgets/auStaticText.h"
+#include "../widgets/BasicMenu.h"
 #include "../widgets/NumericTextCtrl.h"
+#include "../widgets/wxWidgetsWindowPlacement.h"
```

Comment:
I am trying for myself a rebasing onto recent master and building each commit.  So you will discover too if you do that, the need to change this path because the file is now in a library.